packages/usa-date-picker/src/test/date-picker-mousemove.spec.js

```diff
@@ -64,32 +64,6 @@ tests.forEach(({ name, selector: containerSelector }) => {
       );
     });
 
-    it("should update the focus when moving over a non-selected valid day", () => {
-      root.dataset.minDate = "2020-06-01";
-      root.dataset.maxDate = "2020-06-24";
-      input.value = "6/20/2020";
-      EVENTS.click(button);
-      assert.strictEqual(
-        getCalendarEl(".usa-date-picker__calendar__date--focused").dataset
-          .value,
-        "2020-06-20",
-        "focuses correct date",
-      );
-
-      EVENTS.mouseover(
-        getCalendarEl().querySelector(
-          '.usa-date-picker__calendar__date[data-day="19"]',
-        ),
-      );
-
-      assert.strictEqual(
-        getCalendarEl(".usa-date-picker__calendar__date--focused").dataset
-          .value,
-        "2020-06-19",
-        "focuses correct date",
-      );
-    });
-
```

Comment:
Removed test because we are no longer setting focus when date buttons are hovered.